staticd/static_nht.c

```diff
@@ -58,7 +58,7 @@ static void static_nht_update_safi(struct prefix *p, uint32_t nh_num,
 			    && si->type != STATIC_IPV4_GATEWAY_IFNAME
 			    && si->type != STATIC_IPV6_GATEWAY
 			    && si->type != STATIC_IPV6_GATEWAY_IFNAME)
-				return;
+				continue;
```

Comment:
1) Please update the commit message to conform to the http://docs.frrouting.org/projects/dev-guide/en/latest/workflow.html#coding-practices-style

2) a continue probably should be replaced on line 55 as well.